### PR TITLE
[E129-MG < T1048-MG] Merge master to Epic and fix differences

### PR DESCRIPTION
**Pull request by BorisTasevski**

### CODE REVIEW

File: src/query/procedure/mg_procedure_impl.cpp [hunk 1]
```diff
@@ -333,17 +338,17 @@ memgraph::query::TypedValue ToTypedValue(const mgp_value &val, memgraph::utils::
       return memgraph::query::TypedValue(std::move(tv_map));
     }
     case MGP_VALUE_TYPE_VERTEX:
-      return memgraph::query::TypedValue(val.vertex_v->impl, memory);
+      return memgraph::query::TypedValue(val.vertex_v->getImpl(), memory);
```
Comment (Josipmrden) [inline src/query/procedure/mg_procedure_impl.cpp:341]: GetImpl, methods are in pascal case

File: src/query/procedure/mg_procedure_impl.cpp [hunk 2]
```diff
@@ -463,12 +468,31 @@ mgp_value::mgp_value(const memgraph::query::TypedValue &tv, mgp_graph *graph, me
     }
     case MGP_VALUE_TYPE_VERTEX: {
       memgraph::utils::Allocator<mgp_vertex> allocator(m);
-      vertex_v = allocator.new_object<mgp_vertex>(tv.ValueVertex(), graph);
+      vertex_v = std::visit(
+          memgraph::utils::Overloaded{
+              [&](memgraph::query::DbAccessor *) { return allocator.new_object<mgp_vertex>(tv.ValueVertex(), graph); },
+              [&](memgraph::query::SubgraphDbAccessor *impl) {
+                return allocator.new_object<mgp_vertex>(
+                    memgraph::query::SubgraphVertexAccessor(tv.ValueVertex(), impl->getGraph()), graph);
```
Comment (Josipmrden) [inline src/query/procedure/mg_procedure_impl.cpp:476]: GetGraph

Comment (BorisTasevski) [reply src/query/procedure/mg_procedure_impl.cpp:476]: I didn't create

File: src/query/procedure/mg_procedure_impl.cpp (hunk 1 above)
Comment (BorisTasevski) [reply src/query/procedure/mg_procedure_impl.cpp:341]: I didn't create the method, only reused it

**Review by Josipmrden — APPROVED** [review]

**Pull request merged**